TundraTD/Assets/Scripts/City/CityGatesUI.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace City
+{
+    /// <summary>
+    /// Represents the UI behaviour of CityGates
+    /// </summary>
+    [RequireComponent(typeof(CityGates))]
+    public class CityGatesUI : MonoBehaviour
+    {
+        [SerializeField] private Image topHealthBar;
+        [SerializeField] private Text influencePointsHolder;
+
+        private void Start()
+        {
+            if (topHealthBar.sprite is null)
+                throw new Exception("No sprite was assigned");
```

Comment:
Please, specify the exception to make more concrete error description in the game
```suggestion
                throw new ArgumentNullException("No sprite was assigned");
```